frontend/src/setupProxy.js

```diff
@@ -8,11 +8,4 @@ module.exports = (app) => {
       changeOrigin: true,
     })
   );
-  app.use(
-    "/public",
-    createProxyMiddleware({
-      target: process.env.REACT_APP_BACKEND_URL,
-      changeOrigin: true,
-    })
-  );
```

Comment:
@tahierhussain QQ: why this is removed? not required anymore?

Reply:
@vishnuszipstack Yes, it is no longer necessary. Initially, I added this when I planned for the SPS APIs to start with `/public`. However, I later changed the prefix to the default one, i.e., `api/v1`, and made the SPS APIs public.